Sources/API/Routes/BalanceRoutes.swift

```diff
@@ -32,9 +32,11 @@ public final class BalanceRoutes {
      Retrieve a balance transaction
      Retrieves the balance transaction with the given ID.
      
+     - parameter transactionId: The ID of the desired balance transaction (as found on any API object that affects the balance, e.g. a charge or transfer).
+     
      - returns: A StripeRequest<> item which you can then use to convert to the corresponding node
      */
-    public func retrieveBalance(forTransaction transactionId: String) throws -> StripeRequest<BalanceTransactionItem> {
+    public func retrieveBalanceTransaction(transactionId: String) throws -> StripeRequest<BalanceTransactionItem> {
```

Comment:
The naming convention here was to make things descriptive in the sense of this: `retrieveBalance(forTransaction ID)` Going with your route is fine, no issue there, but you have a duplicate descriptor here `retrieveBalanceTransaction(transactionId: ID)`. If we are going to use the function name as the descriptor, lets omit the descriptor as a parameter name. Define that method like `retrieveBalanceTransaction(_ transactionId: String)`